config/locales/views/circuitverse/ar.yml

```
@@ -0,0 +1,154 @@
+    featured_examples:
+        main_heading: "أمثلة مميزة"
+        main_description: "تعلم واستلهم من أفضل الأمثلة"
+        circuit1_text: "مُجمّع حمل التموج 16 بت"
+        circuit2_text: "SAP-1"
+        circuit3_text: "جامع كامل من جامعتي نصف"
+    explore_more: "استكشاف المزيد"
+    editor_picks:
+    main_heading: "اختيارات المحرر"
+    main_description: "شاهد أفضل الدوائر المختارة من قبل المحرر"
+    recent_circuits:
+    main_heading: "الدوائر الحديثة"
```

YAMLlint (1.35.1)

[error] duplication of key "main_heading" in mapping

(key-duplicates)